### PR TITLE
refactor: adjustments with get country method

### PR DESCRIPTION
**Pull request by aureate-labs-team**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/composables/src/getters/cartGetters.ts [hunk 1]
```diff
@@ -36,8 +33,7 @@ export const getCartItemPrice = (product: any): AgnosticPrice => {
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const getCartItemQty = (product: LineItem): number => product?.quantity;
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export const getCartItemAttributes = (product: LineItem, filterByAttributeName?: Array<string>) => {
+export const getCartItemAttributes = (product: LineItem) => {
```
Comment (jeffpdotone) [inline packages/composables/src/getters/cartGetters.ts:36]: Let's refactor this one too moving away from using getters and directly using the raw response.

File: packages/composables/src/useCart/index.ts [hunk 2]
```diff
@@ -36,11 +38,12 @@ const params: UseCartFactoryParams<Cart, CartItem, Product> = {
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   addItem: async (context: Context, { currentCart, product, quantity, customQuery }) => {
-    const appKey = context.$shopify.config.app.$config.appKey;
+    const APP = context.$shopify.config.app;
```
Comment (jeffpdotone) [inline packages/composables/src/useCart/index.ts:41]: please use camelCase here to be consistent.

File: packages/composables/src/getters/cartGetters.ts (hunk 1 above)
Comment (aureate-labs-team) [reply packages/composables/src/getters/cartGetters.ts:36]: I'll create a new task for this. As right now it's using the formatSelectedAttributeList() method, which is also used by another getter method getVariantByAttributes() is using it. So keep it as it is for now. will be refactor in next PR :)

**Review by jeffpdotone — CHANGES_REQUESTED** [review]
Please add unit tests for the methods being modified here. Can be just a happy path of a function been called with correct parameters.

**Comment by jeffpdotone** [comment]
Please resolve conflicts.

**Pull request merged**